resources/mobile/learning-group/session-two.md

```diff
@@ -0,0 +1,191 @@
+# iOS Learning Group Session Two
+
+## Homework review
+
+- iOS Software is really different from web software: we ship a binary instead of serving a web page. - What other
+  kinds of software have the same constraints? - We talked today about the disadvantages of iOS' approach to
+  deployments, but can you think of any benefits?
+- Come up with an idea for Artsy's iOS app. A feature, bug fix, a whole new screen, whatever! Think about how and
+  where the code would be to implement it.
+
+## Exercise: Let's pull latest changes!
+
+Over the past week, there have been changes to both the Eigen and Emission repositories. Let's pull from each and
+make sure we're able to launch the latest code in our simulators.
+
+### Eigen
+
+Remember that Eigen uses CocoaPods, which is itself managed through Bundler. You might need to update your gems
+_first_ and then your CocoaPods.
+
+```sh
+# cd into eigen directory
+git pull
+bundle install # if necessary
+bundle exec pod install # might need to add --repo-update if it fails
+# Open workspace
+```
+
+### Emission
+
+Emission has [pull-lock](https://github.com/orta/pull-lock) configured, but can still get into problems. Here're
+the manual steps in case it fails:
+
+```sh
+# cd into emission directory
+git pull
+yarn install # include --check-files if things get weird
+cd Example
+bundle exec pod install # might need to add --repo-update if it fails
+# Open workspace
+cd ..
+yarn start
+```
+
+## Why React Native?
+
+Artsy uses React Native because it
+[allows Artsy engineers working on the iOS and web front-ends to share the same idioms and tooling](https://artsy.github.io/blog/2019/03/17/three-years-of-react-native/).
+The primary benefit of using React Native is that
+[we get to use React](https://ashfurrow.com/blog/the-case-for-react-native/#react-itself-is-amazing), which is not
+only a better way of building iOS apps, but gives us those shared idioms and tools. (We're hoping to leverage React
```

Comment:
It might be worth looking into the code for one of the components in palette that has a web version and react native version. Maybe the button component?